django/utils/translation/trans_real.py

```diff
@@ -103,6 +103,7 @@ class DjangoTranslation(gettext_module.GNUTranslations):
     def __init__(self, language):
         """Create a GNUTranslations() using many locale directories"""
         gettext_module.GNUTranslations.__init__(self)
+        self.set_output_charset('utf-8')
```

Comment:
should we add a comment "# For Python 2 only" so we can remove it when dropping that?